### ISSUE TITLE
Keyboard navigation in the merge cells demo

### ISSUE THREAD

**Issue by jansiegel** [handsontable/handsontable#2093]
Keyboard navigation does not work properly in merge cells demo - cannot get to the far left side of the table.

-------

##### Steps to reproduce:
* Go to http://handsontable.com/demo/merge_cells.html
* Scroll the table to the right, so that only the second merge group is visible
* Click on `F6`
* Press `left arrow` multiple times

##### Result:
The table stops with the `B` column on its left side, the selection goes 'under' the container

**Comment by jansiegel** [comment]
Released in version 0.12.6
